openapi.yaml

```diff
@@ -1,279 +1,662 @@
+              - nauticalmiles
+        - in: query
+          name: filter[isOpen]
+          description: "Set to true to only search for open restaurants/buildings. Default: false to search for all the results"
+          required: false
+          schema:
+            type: boolean
+        - in: query
+          name: filter[campus][oneOf]
+          description: "Name of the campus. Valid options are: corvallis, extension, cascades, hmsc, and other."
+          required: false
+          schema:
+            type: array
+            items:
+              type: string
+              enum:
+                - corvallis
+                - extension
+                - cascades
+                - hmsc
+                - other
+        - in: query
+          name: filter[type]
+          description: "Type of location. Valid options are: building, dining, cultural-center, parking, and other."
+          schema:
+            type: array
+            items:
+              type: string
+              enum:
+                - building
+                - dining
+                - cultural-center
+                - parking
+                - other
+        - in: query
+          name: filter[giRestroom]
+          description: >
+            If true, locations without gender inclusive restrooms will be filtered out.
+          required: false
+          schema:
+            type: boolean
+        - in: query
+          name: filter[parkingZoneGroup][oneOf]
+          description: >
+            Locations in results should match the parkingZoneGroup
+            in this query parameter.
+          required: false
+          schema:
+            type: array
+            items:
+              type: string
+              # what are the possible values for this?
+        - in: query
+          name: filter[adaParkingSpaceCount][gte]
+          description: "Only include locations with ADA parking space greater than or equal to this amount"
+          required: false
+          schema:
+            type: integer
+        - in: query
+          name: filter[motorcycleParkingSpaceCount][gte]
+          description: "Only include locations with motorcycle parking space greater than or equal to this amount"
+          required: false
+          schema:
+            type: integer
+        - in: query
+          name: filter[evParkingSpaceCount][gte]
+          description: "Only include locations with electric vehicle parking space greater than or equal to this amount"
+          required: false
+          schema:
+            type: integer
+        - in: query
+          name: filter[abbreviation]
+          description: "Search by building abbreviation"
+          required: false
+          schema:
+            type: string
+        - in: query
+          name: page[size]
+          description: "Number of results to return. Used in pagination."
+          required: false
+          schema:
+            type: integer
+            maximum: 10000
```

Comment:
I don't think it's a good idea to return 10,000 results in a response. Let's use the same default/max/min pagination value with other APIs: https://github.com/osu-mist/terms-api/blob/752ea46da0045124953a6d08df2ff5b32e16a517/openapi.yaml#L143-L159

Reply:
That sounds good. We don't even have 10,000 of any of the resources in this API, right?